src/screens/Settings/index.tsx

```diff
@@ -478,7 +543,7 @@ function Settings() {
               setAltLegendaryBin={setAltLegendaryBin}
               altGogdlBin={altGogdlBin}
               setAltGogdlBin={setAltGogdlBin}
-              settingsToSave={settingsToSave}
+              settingsToSave={settingsToSave as AppSettings}
```

Comment:
One small thing though, why do we need this type conversion?
Have you check if the buttons to copy the settings are working fine?

Reply:
Tested here, they are working fine

Reply:
the problem is that I set the default state value as `{}` and typescript (or the linter?) complains that it doesn't have the expected properties fron an AppSettings, so forcing it solves the warning, another option is to say allow 2 types of data in the AdvancedSettings component I guess? I don't know if typescript likes `{}` in a type